### PR TITLE
[mcp23017] Added hint to the wiringPi library

### PR DESCRIPTION
**Pull request by mrumpf**
Without this dependency the pi4j library did not work on my Raspberry Pi Model 3 B+.
Unfortunately there is another issue https://github.com/Pi4J/pi4j/issues/319 which can be resolved by updating to pi4j release 1.2 (not yet released)

### CODE REVIEW

File: addons/binding/org.openhab.binding.mcp23017/README.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 
 This binding allows you to have native access for MCP23017 I/O expander on I2C bus.
-It was tested with Raspberry PI 2 and Raspberry PI 3, but probably should work with other devices supported by pi4j library.
+It was tested with Raspberry PI 2 and Raspberry PI 3, but probably should work with other devices supported by [pi4j](http://pi4j.com/) library.
 
 On Raspberry PI user on which openHAB is running (default user name is "openhab") needs to be added to groups "i2c" and  "gpio".
 
+## Dependencies
+
+Make sure that the [wiringPi](http://wiringpi.com/) library has been installed and that the `gpio` command line tool is available to OpenHAB. The shared library `libwiringPi.so` is required by the pi4j Java library to access the GPIO ports. Without satisfying this dependency you will see strange `NoClassDefFoundError: Could not initialize class ...` errors in the OpenHAB logs.
```
Comment (Confectrician) [inline addons/binding/org.openhab.binding.mcp23017/README.md]: Please break down to one line per sentence.

**Comment by cweitkamp** [comment]
@mrumpf I can see your commit, but it does not appear in the [Changes](https://pullapprove.com/openhab/openhab2-addons/pull-request/4806/) section. Do you know what went wrong? Did you delete your branch?

@openhab/2-x-add-ons-maintainers Any ideas how we can fix this? Shall we close this and open a new PR?

**Comment by Hilbrand** [comment]
It looks like the original branch was removed and a pull request with the changes was created against the repository of the original pr and not this repository. I don't know if it works to create a branch with the original name in the repository this PR was created, otherwise a new PR should be created.

**Comment by cweitkamp** [comment]
Superseded by #5000 .

**Pull request closed without merging**